src/architect/account/account.utils.ts

```diff
@@ -39,6 +39,10 @@ export default class AccountUtils {
   }
 
   static async getAccount(app: AppService, account_name?: string, options?: { account_message?: string, ask_local_account?: boolean }): Promise<Account> {
+    if (account_name === 'dev') {
```

Comment:
Instead of doing the comparison here there should be a overloaded function in AccountUtils that takes in an account name and returns if it is a dev account.
Local accounts are kind of their own thing and we try to keep the special logic associated with determining what they are isolated to that utils class.